plugins/lxd/lxd_disk

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+
+import sys
+from pylxd import api
+
+c=api.API()
+
+if len(sys.argv) == 2:
+    if sys.argv[1]=="autoconf":
```

Comment:
There should be a check before autoconf answers "yes", and it should default to "no".  A good check is the API responding, or a required command is present in the path.